docs/rfcs/0002-storing-course-data.md

```diff
@@ -0,0 +1,134 @@
+## Storing course data
+
+### Abstract
+
+The aim of this RFC is to make decisions about the following:
+- The model definitions* that will be needed to store course data.
+- The plan for loading legacy course data, and keeping that data current
+  after we switch to using our own Open edX instance.
+
+Out of scope for this RFC:
```

Comment:
Since you defined the scope above , everything else is out of scope :-)  so maybe remove this paragraph

Reply:
If it's all the same to you, I'd rather leave it here just to be explicit about it